requirements.txt

```diff
@@ -5,6 +5,8 @@ torch>=1.6.0
 srsly>=2.4.0,<3.0.0
 dataclasses>=0.6,<1.0; python_version < "3.7"
 spacy-alignments>=0.7.2,<1.0.0
+# TODO for PR, remove before merge
+git+https://github.com/polm/thinc.git@fix/resizable-initialized-check
```

Comment:
Instead of doing this, we should use the [Explosion bot](https://github.com/explosion/spaCy/blob/master/extra/DEVELOPER_DOCS/ExplosionBot.md) to run tests while specifying a specific Thinc branch

Reply:
At least - assuming the bot works here on `spacy-transformers`. If not, we should extend it to cover this repo as well.

Reply:
The bot does work with this repo, though there was a bug that was causing an issue. That's been resolved now, and this change is gone.